imports/ui/unit-explorer/units-by-roles.jsx

```diff
@@ -1,50 +0,0 @@
-import React, { Component } from 'react'
```

Comment:
It's not a big issue and doesn't require undoing, but for future changes, please try renaming a file instead of deleting and copying it so we could keep the full git history of it.